examples/shadertoy.rs

```diff
@@ -4,9 +4,17 @@ use macroquad::megaui::widgets::{Label, TreeNode};
 
 use glam::vec3;
 
-#[macroquad::main("Shadertoy")]
+fn window_conf() -> Conf {
+    Conf {
+        window_title: "Shadertoy".to_string(),
+        fullscreen: true,
+        ..Default::default()
+    }
+}
+
+#[macroquad::main(window_conf)]
 async fn main() {
-    let ferris = load_texture("rust.png").await;
+    let ferris = load_texture("examples/rust.png").await;
```

Comment:
IIRC examples/rust.png will not work on wasm - there is no examples folder for textures

Reply:
In this case have to run executables from examples directory.
Otherwise just run `cargo run --example shadertoy` works correctly

Reply:
Also there an issue of this problem https://github.com/not-fl3/macroquad/issues/26